src/client.js

```diff
@@ -81,7 +83,10 @@ export class Saturn {
       }
     }
 
-    const origins = options.origins
+    let origins = options.origins
+    if (!origins || origins.length === 0) {
+      origins = [options.url] ?? [options.cdnURL]
```

Comment:
`[options.url]` won't ever be null/undefined so `[options.cdnURL]` is a dead path

Reply:
updated again